src/main/target/common_pre.h

```diff
@@ -137,6 +137,14 @@
 #define DEFAULT_AUX_CHANNEL_COUNT       6
 #endif
 
+// Set the default cpu_overclock to the first level (108MHz) for F411
+// Helps with looptime stability as the CPU is borderline when running native gyro sampling
+#if defined(USE_OVERCLOCK) && defined(STM32F411xE)
+#define DEFAULT_CPU_OVERCLOCK 1
```

Comment:
Not super stoked about enabling overclocking by default, but I guess there is no other practicable way to get these changes and still support F411...

Reply:
I agree and tried everything I could to avoid it. The F411 was the reason I had to put so much effort into the scheduler as it wasn't even close to being able to run when I started. Even with all the optimizations the combination of the gyro sample, PID and RX (when they align) works out to needing about 131us on average. So this presents a problem when you're trying to stay on a 125us schedule. The same task load with the lowest overclock results in an average of 111us so we just step back into stability.

Reply:
Thanks for exploring the options.